src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1055,13 +1066,31 @@ private Task WritePersonalizedFilesToStorageAccountAsync(IStorageAccount storage
                         new Utility.ConfigReplaceTextItem("{DefaultStorageAccountName}", configuration.StorageAccountName),
                         new Utility.ConfigReplaceTextItem("{ManagedIdentityName}", managedIdentityName)
                     }, "scripts", ContainersToMountFileName));
+
+                    // Configure Cromwell config file for Docker Mysql or PostgreSQL on Azure.
```

Comment:
I know this isn't the most elegant solution, but given how diverged the database section of the conf file is I felt this was the easiest readable, least likely to run into formatting issues way to approach the replacement. Honestly this code will get ripped out anyway when we deprecate Docker Mysql in 4.0.

Reply:
I'd recommend an if statement here instead of using the List<string>.  Also there are minor style issues (need blank white line before and after if statement block)

```
if (configuration.ProvisionPostgreSqlOnAzure.GetValueOrDefault())
{
// TODO fill out the proper string values below
                    await UploadTextToStorageAccountAsync(storageAccount, ConfigurationContainerName, CromwellConfigurationFileName, Utility.PersonalizeContent(new[]
                    {
                        new Utility.ConfigReplaceTextItem("{DatabaseUrl}", ""),
                        new Utility.ConfigReplaceTextItem("{DatabaseUser}",  ""),
                        new Utility.ConfigReplaceTextItem("{DatabasePassword}",  ""),
                        new Utility.ConfigReplaceTextItem("{DatabaseDriver}", ""),
                        new Utility.ConfigReplaceTextItem("{DatabaseProfile}", ""),
                    }, "scripts", CromwellConfigurationFileName));
}
else
{
// TODO fill out the proper string values below
                    await UploadTextToStorageAccountAsync(storageAccount, ConfigurationContainerName, CromwellConfigurationFileName, Utility.PersonalizeContent(new[]
                    {
                        new Utility.ConfigReplaceTextItem("{DatabaseUrl}",  ""),
                        new Utility.ConfigReplaceTextItem("{DatabaseUser}", ""),
                        new Utility.ConfigReplaceTextItem("{DatabasePassword}", ""),
                        new Utility.ConfigReplaceTextItem("{DatabaseDriver}",  ""),
                        new Utility.ConfigReplaceTextItem("{DatabaseProfile}",  ""),
                    }, "scripts", CromwellConfigurationFileName));
}

```

Reply:
I personally find that a lot harder to read and didn't love the idea of the duped code but if you think that'd be easier to understand i'll make the change.

Reply:
updated

Reply:
> I personally find that a lot harder to read and didn't love the idea of the duped code but if you think that'd be easier to understand i'll make the change.

Good idea, could refactor out the "UploadTextToStorageAccountAsync()" call so it's not duplicated